### PR TITLE
add description for formality classifier

### PR DESCRIPTION
**Pull request by sweta20**
*Description of changes: Adds a description of the Formality Classifier (to be released) with usage instructions.*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: IWSLT2023/README.md [hunk 1]
```diff
@@ -42,6 +42,20 @@ where the target language `tl` is one of: `ko, vi, pt, ru`; \
 domain refers to the data sources and is one of: `topical-chat`, `telephony`; \
 formality level is one of: `formal` or `informal`.
 
+### Additional Resources
+
+We release a multilingual classifier trained to predict the formality of a text for the language pairs: EN-KO, EN-VI, EN-RU and EN-PT. We finetune [xlm-roberta-base](https://huggingface.co/xlm-roberta-base) model on human-written formal and informal text following the setup from [Briakou et al., EMNLP 2021](https://aclanthology.org/2021.emnlp-main.100.pdf). The model can be used as follows (after installing [transformers](https://pypi.org/project/transformers/) and [datasets](https://pypi.org/project/datasets/)):
```
Comment (xingniu) [inline IWSLT2023/README.md:47]: Is `datasets` used in the code snippet?

File: IWSLT2023/README.md [hunk 2]
```diff
@@ -42,6 +42,20 @@ where the target language `tl` is one of: `ko, vi, pt, ru`; \
...
+
+```python
+from transformers import AutoTokenizer, AutoModelForSequenceClassification, TextClassificationPipeline
+import torch
+model_name = "xlm-roberta-base"
+tokenizer = AutoTokenizer.from_pretrained(model_name)
+model = AutoModelForSequenceClassification.from_pretrained(<path to the downloaded model directory>)
```
Comment (xingniu) [inline IWSLT2023/README.md:54]: Where can we download the model?

File: IWSLT2023/README.md [hunk 3]
```diff
@@ -42,6 +42,20 @@ where the target language `tl` is one of: `ko, vi, pt, ru`; \
...
+pipe = TextClassificationPipeline(model=model, tokenizer=tokenizer)
+scores = pipe(text, return_all_scores=True)
```
Comment (xingniu) [inline IWSLT2023/README.md:56]: What is the expected score range? Which end corresponds to formal/informal?

File: IWSLT2023/README.md (hunk 1 above)
Comment (sweta20) [reply IWSLT2023/README.md:47]: It is not, I will remove it.

File: IWSLT2023/README.md (hunk 2 above)
Comment (sweta20) [reply IWSLT2023/README.md:54]: I am still working with Ben on adding it as a release. I do not have the right permissions.

File: IWSLT2023/README.md (hunk 3 above)
Comment (sweta20) [reply IWSLT2023/README.md:56]: I will add a note. The scores are probabilities (0-1) with 1 corresponding to output being formal.

**Comment by xingniu** [comment]
LGTM. Please update the instruction of downloading your model when it's ready.

**Pull request merged**